xbmc/utils/XMLUtils.h

```diff
@@ -20,6 +21,7 @@ class XMLUtils
 {
 public:
   static bool HasChild(const TiXmlNode* pRootNode, const char* strTag);
+  static bool HasChild(const tinyxml2::XMLNode* pRootNode, const char* strTag);
```

Comment:
this is probably annoying as hell but while at it can you also rename the variables on the methods signature to conform to the current code guidelines? eg: `rootNode` and `tag`

Reply:
Yeah, that sucked. But its done. I'll leave everything as fixups for now. Have only touched tinyxml2 methods, as the intent would be to remove the tinyxml1 methods eventually obviously.